### PR TITLE
PAR - Thibaut Davoult - it2 done but need to loop on buttons

### PR DESCRIPTION
**Pull request by thibautdavoult**

### CODE REVIEW

**Comment by thibautdavoult** [comment]
cc @stuartogardner

File: starter-code/app.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Vue.component("bulma-message", {
```
Comment (yacinehmito) [inline starter-code/app.js:1]: Components should be named with PascalCase. Here it should be `BulmaMessage`.

File: starter-code/app.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <button class="delete" @click="showMessage = !showMessage" aria-label="delete"></button>
+    </div>
+    <div v-show="showMessage" class="message-body">
+      {{ body }}
+    </div>
+  </article>
+  `
+})
+
+Vue.component("bulma-modal", {
```
Comment (yacinehmito) [inline starter-code/app.js:21]: Components should be named with PascalCase. Here it should be `BulmaModal`.

File: starter-code/app.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  props: ['head', 'body', 'show'],
+  template: `
+  <div class="modal is-active" v-show="show">
+    <div class="modal-background"></div>
+    <div class="modal-card">
+      <header class="modal-card-head">
+        <p class="modal-card-title"> Country is: {{ head }}</p>
```
Comment (yacinehmito) [inline starter-code/app.js:28]: You probably don't want hardcoded text here, because it makes the component less reusable.

**Review by yacinehmito — APPROVED** [review]
I am going to review each iteration individually:

1. **message:** Very good (although you didn't check whether the color was valid)
2. **modal:** Excellent!
3. **breadcrumbs:** Not relevant because you didn't do it.

All in all, very good work. Keep it up! 💯

**Pull request closed without merging**